releasenotes/notes/add-commutation-library-88b7ff65b3d35f9a.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Adds a commutation library to the :class:`CommutationChecker`. The commutation library stores all
+    commutation relations of unparameterizable standard gates into a dictionary that allows for efficient
+    lookup at runtime. Furthermore, the :class:`CommutationChecker` was refactored and an upper limit was
+    set to the number of cached commutation relations that are not in the commutation library. A session
```

Comment:
To me it seems that adding a commutation library to the commutation checker class and adding a session commutation checker are somewhat different features, so I think it's best to separate the text into multiple features, as in:
```
features:
  - |
    feature 1 text
  - |
    feature 2 text
```

Reply:
Agreed.